pkgs/development/ocaml-modules/junit/ounit.nix

```diff
@@ -3,7 +3,9 @@
 buildDunePackage ({
   pname = "junit_ounit";
 
-  inherit (junit) src version meta useDune2;
+  inherit (junit) src version meta;
+
+  duneVersion = "3";
```

Comment:
inherit

Reply:
Have you tested to inherit? I think it will fail.

Reply:
Why would it fail? We were doing the same to useDune2 before?

Reply:
Because of: https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/ocaml/dune.nix#L42

Reply:
Very well I see.